From 5d678836f4baff50b8e4e202c097039c333b585c Mon Sep 17 00:00:00 2001
From: Paul Reioux <reioux@gmail.com>
Date: Mon, 24 Jun 2013 01:17:53 -0500
Subject: [PATCH 190/507] Revert "msm: pm-data: Enable
 powercollapse/suspend_enabled for non-boot cpus"

This reverts commit d5ac51b7bd43225ecfaf57dcc7a0564f48cb6249.

Signed-off-by: Simarpreet Singh <simar@linux.com>
---
 arch/arm/mach-msm/pm-data.c |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-msm/pm-data.c b/arch/arm/mach-msm/pm-data.c
index f41c569..fa43982 100644
--- a/arch/arm/mach-msm/pm-data.c
+++ b/arch/arm/mach-msm/pm-data.c
@@ -1,4 +1,4 @@
-/* Copyright (c) 2012-2013, The Linux Foundation. All rights reserved.
+/* Copyright (c) 2012, The Linux Foundation. All rights reserved.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License version 2 and
@@ -46,7 +46,7 @@ struct msm_pm_platform_data msm_pm_sleep_modes[] = {
 		.idle_supported = 0,
 		.suspend_supported = 1,
 		.idle_enabled = 0,
-		.suspend_enabled = 1,
+		.suspend_enabled = 0,
 	},
 
 	[MSM_PM_MODE(1, MSM_PM_SLEEP_MODE_POWER_COLLAPSE_STANDALONE)] = {
@@ -74,7 +74,7 @@ struct msm_pm_platform_data msm_pm_sleep_modes[] = {
 		.idle_supported = 0,
 		.suspend_supported = 1,
 		.idle_enabled = 0,
-		.suspend_enabled = 1,
+		.suspend_enabled = 0,
 	},
 
 	[MSM_PM_MODE(2, MSM_PM_SLEEP_MODE_POWER_COLLAPSE_STANDALONE)] = {
@@ -102,7 +102,7 @@ struct msm_pm_platform_data msm_pm_sleep_modes[] = {
 		.idle_supported = 0,
 		.suspend_supported = 1,
 		.idle_enabled = 0,
-		.suspend_enabled = 1,
+		.suspend_enabled = 0,
 	},
 
 	[MSM_PM_MODE(3, MSM_PM_SLEEP_MODE_POWER_COLLAPSE_STANDALONE)] = {
-- 
1.7.9.5

